vcluster/_partials/manage/deploy-changes.mdx

```diff
@@ -95,22 +95,36 @@ Replace:
 Learn more about [Cluster API Provider for vCluster](https://github.com/loft-sh/cluster-api-provider-vcluster).
 
 <Flow>
+<Step>
+
+   [Install](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) the `clusterctl` CLI.
+
+</Step>
+<Step>
+
+   Install the vCluster provider.
+
+   ```bash
+   clusterctl init --infrastructure vcluster:v0.2.0
```

Comment:
Do you need to do this step for an already existing vcluster?

Reply:
You will need first two steps (this and the previous one), say I have deployed a cluster, and you are trying to update the vcluster version. And you don't have these installed. (atleast that is my understanding)

Reply:
I could see leaving this in if they somehow changed their `clusterctl` setup.